tests/unit_tests/test_public_api.py

```diff
@@ -66,7 +66,7 @@ def test_parse_path_docker_proj(mock_server, api):
     assert r == "run"
 
 
-def test_parse_path_url(api):
+def test_parse_path_url(api, mock_server):
```

Comment:
Turns out all the `_parse_path` implementations call `.default_entity` this was calling out to `https://api.wandb.ai` and I saw a flake where we returned a 500 😭 .  Makes me think we somehow aren't retrying these `.default_entity` calls which isn't great, but we shouldn't be hitting prod from our tests.  Begs the question of what other tests are calling out, but we'll save that for another day.